### PR TITLE
Geocoding Firms and Subsidiaries

### PR DESCRIPTION
**Pull request by lshepstone**
This is dependent on the work being done in the `geocoding` branch for `mas-rad_core`.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ gem 'dough-ruby',
   ref: 'cf08913'
 gem 'kaminari'
 gem 'letter_opener', group: :development
-gem 'mas-rad_core'
+gem 'mas-rad_core', github: 'moneyadviceservice/mas-rad_core', branch: 'geocoding'
```
Comment (lshepstone) [inline Gemfile]: This will be reset once https://github.com/moneyadviceservice/mas-rad_core/pull/5 is pulled in.

File: spec/support/geocoder.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require 'geocoder'
+
+Geocoder.configure(:lookup => :test)
```
Comment (benlovell) [inline spec/support/geocoder.rb]: Old school hash rocket.

**Comment by benlovell** [comment]
Provisional :+1: 

The history might be a little more cohesive.

**Comment by lshepstone** [comment]
History cleaned up @benlovell

**Comment by alexwllms** [comment]
LGTM :+1:

**Comment by benlovell** [comment]
:ship:

**Pull request merged**